eyeshade/migrations/0001_transactions/up.sql

```diff
@@ -2,8 +2,8 @@ select execute($$
 
 insert into migrations (id, description) values ('0001', 'transactions');
 
-create type account_type as enum ('channel', 'owner', 'uphold', 'internal');
-create type transaction_type as enum ('contribution', 'referral', 'contribution_settlement', 'referral_settlement', 'fees', 'scaleup', 'manual');
+create type account_type as enum ('channel', 'owner', 'uphold', 'internal', 'paymentId');
+create type transaction_type as enum ('contribution', 'referral', 'contribution_settlement', 'referral_settlement', 'fees', 'scaleup', 'manual', 'ad_payout');
```

Comment:
we need two types, one into the `paymentId` account and one out. I suggest `ad` and `ad_settlement` for consistency (also `ad_payout` and `ad_settlement` would be confusing)